tests/data/cli/validate_scenarios/definitions/region/regions.yaml

```diff
@@ -0,0 +1,4 @@
+- common:
```

Comment:
Can you use an existing definitions-folder in the test-data? We're adding quite a lot of very similar items here and it's hard to keep an overview...